src/ghost-content/index.js

```diff
@@ -141,39 +143,68 @@ module.exports = ({ logger, db, projectLocation, enabled }) => {
 
   const updatePendingForAllFolders = () => Promise.each(trackedFolders, updatePendingForFolder)
 
-  const recordRevision = async (rootFolder, file, content) => {
+  const recordRevision = (knex, content_id, trx) =>
+    knex('ghost_revisions')
+      .transacting(trx)
+      .insert({ content_id, revision: uuid.v4(), created_by: 'admin' })
+
+  const upsertFile = async (rootFolder, file, content) => {
     const knex = await db.get()
 
-    const { normalizedFolderName } = normalizeFolder(rootFolder)
+    const folder = normalizeFolder(rootFolder).normalizedFolderName
+
+    if (
+      await knex('ghost_content')
+        .where({ folder, file, content })
+        .count('id')
+        .then(([res]) => Number(res.count) > 0)
```

Comment:
this query is better but I have a question — why do you need to convert it to number?
also shouldn't it be `count('id as count')` to work?

Reply:
`count('id')` works fine. As of [documentation](http://knexjs.org/#Builder-count):

> count returns a bigint type which will be a String and not a Number (more info).

Reply:
ah, because of Postgres. Can you please then also make such fixes to `botpress/src/content/service.js`? It uses count in 2 places without converting it to number

Reply:
Done!